.github/CODEOWNERS

```diff
@@ -1007,7 +1007,7 @@ sdk/ai/ai-projects @ganeshyb @GraceBrigham
 /sdk/monitor/monitor-opentelemetry @hectorhdzg @JacksonWeber @harsimar @MSNev
 /sdk/monitor/monitor-opentelemetry-exporter @hectorhdzg @JacksonWeber @harsimar @MSNev
 /sdk/monitor/monitor-query/ @srnagar @lmolkova @Azure/azure-sdk-write-monitor-data-plane
-/sdk/monitor/perf-tests/ @srnagar @Azure/azure-sdk-write-monitor-data-plane
+/sdk/monitor/monitor-*-perf-tests/ @srnagar @Azure/azure-sdk-write-monitor-data-plane
```

Comment:
orthogonal but why are the perf tests at the same level as the package instead of under the tests folder?

Reply:
We recently adjust the perf-tests directory structure to suit `pnpm` workspaces better.  We are moving from rush to pnpm. @mpodwysocki may have more details.

Reply:
/cc @mpodwysocki who is driving the pnpm work